src/app/metadata/features/global-summary/global-summary.component.ts

```diff
@@ -47,5 +48,34 @@ export class GlobalSummaryComponent {
   datasets = computed(() => this.stats().Dataset);
   methods = computed(() => this.stats().ExperimentMethod);
   individuals = computed(() => this.stats().Individual);
-  files = computed(() => this.stats().ProcessDataFile);
+  files: Signal<FileStats> = computed(() => this.#getFiles());
+
+  /**
+   * Aggregates file statistics from the individual file stats in the global summary.
+   * @returns The aggregated file statistics from all file stats.
+   */
+  #getFiles(): FileStats {
+    let fileCount = 0;
+    const formats = new Map();
+    Object.entries(this.stats())
+      .filter(([key]) => key.endsWith('File'))
+      .forEach(([, fileSummary]) => {
+        fileCount += fileSummary.count;
+        fileSummary.stats.format.forEach((x: ValueCount) => {
+          let found = formats.get(x.value);
+          if (found) {
+            formats.set(x.value, x.count + found);
+          } else {
+            formats.set(x.value, x.count);
+          }
+        });
+      });
+    const formatsDict: ValueCount[] = Array.from(formats.entries()).map(
+      ([value, count]) => ({ value, count }),
+    );
+    return {
+      count: fileCount,
+      stats: formatsDict.length > 0 ? { format: formatsDict } : undefined,
+    };
```

Comment:
```suggestion
    let totalCount = 0;
    const totalFormats: Map<string, number> = new Map();
    Object.entries(this.stats())
      .filter(([key]) => key.endsWith('File'))
      .forEach(([, value]) => {
        const fileStats = value as FileStats;
        totalCount += fileStats.count;
        const format = fileStats.stats?.format || [];
        format.forEach(({ value, count }) =>
          totalFormats.set(value, (totalFormats.get(value) || 0) + count),
        );
      });
    const format: ValueCount[] = Array.from(totalFormats.entries()).map(
      ([value, count]) => ({ value, count }),
    );
    return {
      count: totalCount,
      stats: format ? { format } : undefined,
    };
```

I simplified it a bit, made the names a bit clearer and added a check for the case that fileStats.stats is undefined.